maptime/glide.yaml

```diff
@@ -0,0 +1,8 @@
+package: github.com/m3db/build-tools/timemaplint
+import:
+- package: github.com/kisielk/gotool
```

Comment:
could you pin the package versions in the glide.yaml to a semver (where available, or sha1 if not)